WcaOnRails/app/views/delegate_reports/_delegate_report.html.erb

```diff
@@ -3,7 +3,7 @@
 <div class="delegate-report">
   <div>
     <strong>Date</strong>
-    <%= wca_date_range(competition.start_date, competition.end_date) %>
+    <%= wca_date_range(competition.start_date, competition.end_date, locale: :en) %>
```

Comment:
I believe so, through [this](https://github.com/viroulep/worldcubeassociation.org/blob/80d01c2e2ee0372fd33491799236d1330be67a1a/WcaOnRails/app/views/competitions_mailer/notify_of_delegate_report_submission.html.erb) mailer!